recipes/usgs-strec/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "usgs-strec" %}
+{% set version = "2.2.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/usgs-strec-{{ version }}.tar.gz
+  sha256: 9517341d44a5d3acaada2492fb9c99128edfbd85b562a909af2ef058932cda52
+
+build:
+  entry_points:
+    - regcalc = strec.bin.regcalc:app
+    - strec_cfg = strec.bin.strec_cfg:app
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python <=3.11
+    - setuptools >=42
+    - wheel
+    - setuptools-scm >=3.4
+    - cython >=0.29.23
+    - numpy
+    - pip
+  run:
+    - python <=3.11
+    - configobj >=5.0.6
+    - esi-utils-rupture
+    - geopy >=2.2.0
+    - numpy >=1.21.0
+    - openpyxl >=3.0.9
+    - pandas >=1.3.5
+    - rasterio >=1.2.10
+    - typer <0.7.0
+    - xlrd >=2.0.1
+    - xlsxwriter >=3.0.1
+    - xlwt >=1.3.0
+
+test:
+  imports:
+    - usgs_strec
```

Comment:
```suggestion
    - strec
```

Reply:
Will do, thanks! This rename is confusing.